Pipfile

```diff
@@ -0,0 +1,48 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[packages]
+
+Django = ">=1.11,<2"
+psycopg2 = ">=2.7.3"
+Pillow = ">=4.2"
+awesome-slugify = "*"
+csscompressor = "==0.9.5"
+django-braces = ">=1.11"
+django-bootstrap-form = "==3.3"
+djangocodemirror = { git = 'https://github.com/sveetch/djangocodemirror.git', ref = 'a2e4920f7483cbd6fb090652ef18d5d3200a6e5d' }
```

Comment:
ĉu djando-codemirror funkcias en 1.11? Dĵango tute ŝanĝis la manieron `widgets` estas generitaj, kaj la malnova maniero estas rompita.

Reply:
Mi fiksis al la antaŭlasta komito, kiun mi verkis. Tiu verŝajne funkcias.
Sed laŭ la aŭtoro, tio rompiĝis la testojn. Do malfaris: https://github.com/sveetch/djangocodemirror/commits/master

Pipenv trairas la dependaĵoj, kaj kun la lasta versio de codemirror, ne eblis instali Dj 1.11.